src/pages/Campaign/index.tsx

```diff
@@ -402,7 +402,9 @@ export default function Campaign() {
                         ? getFormattedTimeFromSecond((selectedCampaign.startTime - now) / 1000)
                         : selectedCampaign.status === 'Ongoing'
                         ? getFormattedTimeFromSecond((selectedCampaign.endTime - now) / 1000)
-                        : 'ENDED'
+                        : new Date(selectedCampaign.endTime).toISOString().slice(0, 10) +
```

Comment:
should left a comment.
```
// '2022-07-28 11:45' 
// 'yyyy-mm-dd hh:mm' 
```
anyway, we have dayjs can format for us.

Reply:
@nguyenhoaidanh fixed.

Reply:
Have you committed?

Reply:
Yeah.